delete identifiers.

linuxlonelyeagle opened the pull request:
see [here](https://github.com/llvm/circt/issues/6610).

cgyurgyik commented on include/circt/Conversion/SCFToCalyx.h:
```diff
@@ -25,6 +25,12 @@ namespace scfToCalyx {
 // used as the Calyx port name.
 static constexpr std::string_view sPortNameAttr = "calyx.port_name";
 
+// goPort ,donePort and resetPort identifie the attributes of the go, done and
```
```suggestion
// the goPort, donePort, and resetPort identify the attributes of the go, done and
```

rachitnigam commented:
@cgyurgyik do you know if we also generate `@clk` signals? Those should also probably be added here?

cgyurgyik commented:
[Here](https://github.com/llvm/circt/blob/18499e954e686847177f544a03579aef3622990c/lib/Dialect/Calyx/Transforms/CalyxHelpers.cpp#L66) and [here](https://github.com/llvm/circt/blob/ba50574867277e843405c5efbc72987540638128/lib/Dialect/Calyx/CalyxOps.cpp) you'll find other string literals that could be replaced by these identifiers. It may be more fitting to place these in a more visible file so that they can be used across passes and ops.

linuxlonelyeagle commented:
Can I merge this branch? It doesn't look like there's anyone else reviewing the code, and I'm actually not sure of the conditions for merging this branch.

rachitnigam commented:
@linuxlonelyeagle @cgyurgyik left a couple of comments. Can you please address them before merging?

linuxlonelyeagle commented:
> @linuxlonelyeagle @cgyurgyik left a couple of comments. Can you please address them before merging?

ok.It may take me a while, I'll address this when I have time.

linuxlonelyeagle commented:
I've moved the variables to CalyxOps.h.The main reason I moved the variables to CalyxOps.h is that a lot of CalyxOp.cpp could use them.I don't know what you think.

rachitnigam reviewed (approved):
LGTM!

linuxlonelyeagle commented:
Earlier I said I went to look for a job, but it didn't actually go well, I'm now going to an internship at an internet company, and I'm actually still very much related to Calyx Dialect. I'll continue to do some stuff on it later on when I have some time, so feel free to let me know if you guys have any other things you want to do.

rachitnigam commented:
Got it @linuxlonelyeagle! I think the best step is to look for open issues marked with the Calyx label and contribute to them. Good luck with the future job search!

linuxlonelyeagle commented:
> Got it @linuxlonelyeagle! I think the best step is to look for open issues marked with the Calyx label and contribute to them. Good luck with the future job search!

Thanks!
